src/util.rs

```diff
@@ -7,3 +7,41 @@ macro_rules! regex {
         &REGEX
     }};
 }
+
+use std::{collections::HashMap, iter::FromIterator};
+
+/// Helper for `translate`, does the same thing as string.maketrans in python
+/// requires that f and t are the same length, or it takes the shorter one
+pub fn maketrans(f: &str, t: &str) -> HashMap<char, char> {
```

Comment:
To match the Python semantics, `maketrans` should either panic or return an `Err` if the character lengths of the two inputs are not equal.